upgrade/steps.go

```diff
@@ -1092,3 +1091,13 @@ var fetchLatestJavaGen = stepv2.Func03("Fetching latest Java Gen", func(ctx cont
 	stepv2.SetLabel(ctx, latestJavaGen.String())
 	return currentJavaGen, latestJavaGen.String(), true
 })
+
+var getUpstreamProviderOrg = stepv2.Func11("Get UpstreamOrg from go.mod", func(ctx context.Context, upstreamMod module.Version) string {
```

Comment:
This function isn't getting anything from a `go.mod` file, it's parsing from `module.Version`. 

```suggestion
var parseUpstreamProviderOrg = stepv2.Func11("Parse Upstream Provider Org", func(ctx context.Context, upstreamMod module.Version) string {
```